packages/ketcher-core/src/application/render/raphaelRender.js

```diff
@@ -96,6 +96,17 @@ Render.prototype.page2obj = function (event) {
   return this.view2obj(pp)
 }
 
+/**
+ * @param {Vec2} coord Vector in Raphael coordinate system,
+ *                     scaled atom's position with offset
+ * @returns {Vec2} Vector in View coordinate system,
+ *                 coordinate relative to viewport (excluding toolbar)
+ */
+Render.prototype.raphael2View = function (coord) {
+  const obj = coord.sub(this.options.offset).scaled(1 / this.options.scale)
```

Comment:
propose to rename 'obj' to 'vec' or 'vector', as it could be a more specific name.

Reply:
The reason why I name it `obj` is the `obj` here has the same meaning as `obj` in `obj2View`. What `obj` means is not clear to me as well. I'll take a closer look into this in issue [#2697](https://github.com/epam/ketcher/issues/2697)

Reply:
I discovered - it is a Vec2 instance, Vec2 - itself is a vector class with some Vector calculus abilities.